MAINTAINERS.md

```diff
@@ -1,18 +1,39 @@
 # Laminas Project Maintainers
 
+Maintainers are trusted contributors who have GitHub's `maintainer` rights to one or more repositories under our GitHub organizations.
+
+Beyond driving fixes/improvements, we expect maintainers to:
+
+- Ensure project continuity, quality, and consistency
```

Comment:
One thing mention in the TSC meeting was the expectation that maintainers would commit themselves to be around. 

So I propose to change this requirement to:

- Be committed to the project to ensure its continuity, quality, and consistency